azkaban-hadoop-security-plugin/build.gradle

```diff
@@ -1,12 +1,20 @@
 apply plugin: 'distribution'
 
+configurations {
+    testCompile.extendsFrom compileOnly
```

Comment:
@HappyRay 
Let me know if we should do probably a 30-min or 1-hour discussion on Gradle some time since a GitHub PR doesn't seem to be well suited for explaining Gradle intricacies.